C301.tex

```diff
@@ -68,24 +71,143 @@
+The result is that documentation is curated from the bottom-up, and metadata curation is the responsibility realm of individual teams for their documentation projects.
+
+As of this writing, the Algolia full-text search only includes documents hosted on \texttt{lsst.io}, but we intend to begin including multi-page guides and even documentation content from other content repositories in the near future.
+
+\section{Developer documentation}
+
+Documentation is an essential ingredient of good developer experience.
+Rubin Observatory Data Management maintains a developer guide (\url{https://developer.lsst.io}) to document and provide standards for all aspects of contributing to the Rubin Observatory codebases.
+With respect to writing documentation, the Rubin Developer Guide provides technical guidance on the documentation tools and markup syntaxes.
+In the Developer Guide, we also document \emph{how} to write documentation, such as the content style and content patterns.
+
+\section{Project configuration}
+
+Documentation tools like Sphinx (\url{https://www.sphinx-doc.org}) can be overwhelming for a casual documentation contributor to set up.
+Not only does Sphinx have a large number of configurations, but it also has an even more expansive ecosystem of extensions.
+To alleviate this concern from individual projects, and also promote standardization across all our documentation projects, Rubin provides the \textit{Documenteer} Python package (\url{https://documenteer.lsst.io}) for our Sphinx configuration, extension dependencies, custom extensions, and theming.
+With a configuration module base, the \texttt{conf.py} files for individual projects is reduced to a single single (this approach is inspired by sphinx-astropy, \citet{thomas_robitaille_2023_8015243}).
+The configuration system uses a TOML file to facilitate a simplified framework for per-project configurations.
+
+% Whatever you abstract, you also own
+
+\label{sec:technotes}
+\section{Technotes}
+
+Looking beyond project documentation, we have developed a notion of ``technotes'' as single-page websites that are deployed from their own GitHub repositories \citep{SQR-000}.
+Technotes can be used for a wide range of purposes, from architectural design proposals, to research and test reports.
+Technotes solve the problem of making internal communications consistent and discoverable.
+Rather than being in wiki, personal Google Docs, email, or Slack, technotes are indexed by the documentation hub.
+Being websites, technotes are also available to the astronomy community, and we intend to make this content indexable with ADS.
+
+We are now providing a Python package called \textit{Technote} (\url{https://github.com/lsst-sqre/technote}) that provides Sphinx theme extensions, and configurations that enable Sphinx to generate these single-page articles.
```

Comment:
I stumbled over the comma here when reading. My brain thought it was introducing a separate clause rather than another item provided by Technote.